packages/components/dropdowns/dropdown-menu/src/dropdown-menu.story.js

```diff
@@ -111,4 +115,72 @@ storiesOf('Components|Dropdowns|DropdownMenu', module)
         </SpacingsStack>
       </DropdownMenu>
     </Section>
-  ));
+  ))
+  .add('DropdownMenu - Reposition menu if necessary', () => {
```

Comment:
I don't think this story is needed for devs (or is it?), but I'm leaving it here for now, in case someone wants to check the positioning-logic in storybook during review.